src/tracer/DOME_tracer.F90

```diff
@@ -354,8 +354,9 @@ subroutine initialize_DOME_tracer(restart, day, G, h, OBC, CS, sponge_CSp, &
     if (OBC%apply_OBC_v) then
       allocate(OBC_tr1_v(G%isd:G%ied,G%jsd:G%jed,nz))
       do k=1,nz ; do j=G%jsd,G%jed ; do i=G%isd,G%ied
-        if (k < nz/2) then ; OBC_tr1_v(i,j,k) = 0.0
-        else ; OBC_tr1_v(i,j,k) = 1.0 ; endif
+        if (k < 2) then ; OBC_tr1_v(i,j,k) = 0.0
```

Comment:
I'm not including this change in the merge because it changes the DOME configuration and seems unrelated to the adding sponges in ALE mode.

TODO: If we decide to generalize DOME to work in general coordinates then need to generalize this tracer injection to work in arbitrary coordinates - it is currently specified in terms of "k".